framework/src/modules/chain/components/storage/entities/account.js

```diff
@@ -0,0 +1,503 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const path = require('path');
+const assert = require('assert');
+const _ = require('lodash');
+const AccountEntity = require('../../../../../components/storage/entities/account');
+
+const defaultCreateValues = {
+	publicKey: null,
+	secondPublicKey: null,
+	secondSignature: false,
+	u_secondSignature: false,
+	username: null,
+	u_username: null,
+	isDelegate: false,
+	u_isDelegate: false,
+	balance: '0',
+	u_balance: '0',
+	missedBlocks: 0,
+	producedBlocks: 0,
+	rank: null,
+	fees: '0',
+	rewards: '0',
+	vote: '0',
+	nameExist: false,
+	u_nameExist: false,
+	multiMin: 0,
+	u_multiMin: 0,
+	multiLifetime: 0,
+	u_multiLifetime: 0,
+};
+
+const readOnlyFields = ['address'];
+
+const dependentFieldsTableMap = {
+	membersPublicKeys: 'mem_accounts2multisignatures',
+	u_membersPublicKeys: 'mem_accounts2u_multisignatures',
+	votedDelegatesPublicKeys: 'mem_accounts2delegates',
+	u_votedDelegatesPublicKeys: 'mem_accounts2u_delegates',
+};
+
+const sqlFiles = {
+	create: 'accounts/create.sql',
+	update: 'accounts/update.sql',
+	updateOne: 'accounts/update_one.sql',
+	delete: 'accounts/delete.sql',
+	resetUnconfirmedState: 'accounts/reset_unconfirmed_state.sql',
+	resetMemTables: 'accounts/reset_mem_tables.sql',
+	increaseFieldBy: 'accounts/increase_field_by.sql',
+	decreaseFieldBy: 'accounts/decrease_field_by.sql',
+	createDependentRecord: 'accounts/create_dependent_record.sql',
+	deleteDependentRecord: 'accounts/delete_dependent_record.sql',
+	delegateBlocksRewards: 'accounts/delegate_blocks_rewards.sql',
+	syncDelegatesRank: 'accounts/sync_delegates_rank.sql',
+	insertFork: 'accounts/insert_fork.sql',
+};
+
+class Account extends AccountEntity {
```

Comment:
That will make it more clear while searching in the code, that this Account Entity is specific extension for chain.